openslides_backend/action/generics.py

```diff
@@ -307,7 +307,7 @@ def delete_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
         """
 
         data = []
-        for instance in payload:
+        for instance in self.update_instances(payload):
```

Comment:
Ah, I thought this would stem from a missing rebase, but it is still missing in the DeleteAction. Please move this to a new BaseAction like `GenericBaseAction` or something like that and remove it in the create and update actions to unify the handling.

Reply:
Worked on unifying this. Add a GenericBaseAction with get_updated_instances() and use it in all Generic Actions.